model/task/task.go

```diff
@@ -2009,24 +2010,15 @@ func (t *Task) GetDisplayStatus() string {
 	if t.DisplayStatus != "" {
 		return t.DisplayStatus
 	}
-	if t.Aborted && t.IsFinished() {
+	if t.Aborted {
```

Comment:
Should this function now also set t.DisplayStatus if it works the same way?

Reply:
Sure - it might be that this should've always needed to been the case.

Reply:
Love this refactor!